v2/devices/mako.yml

```diff
@@ -12,6 +12,10 @@ user_actions:
     title: "Reboot to Bootloader"
     description: "Press and hold the volume buttons and the power button until the phone reboots."
     button: true
+  confirm_model:
```

Comment:
You also need to include that message somewhere, e.g. in the unlock steps array.

Reply:
```suggestion
  partition_layout:
```